### ISSUE TITLE
357 AmmoType changes SWEP's force.

### ISSUE THREAD

**Issue by Nitestokeh** [Facepunch/garrysmod-issues#2199]
Greetings, I've found out a problem related to .357 ammo type recently. All other ammo types don't do anything to force, but .357 ammo increases it to 10+ approximately. That wouldn't be a big problem if it didn't send props and ragdolls flying across the map. If it's possible to fix it, that'd help a lot. Thanks in advance.

**Comment by Kefta** [comment]
I thought all HL2 weapons got their forced knocked back up by _Kilburn

**Comment by robotboy655** [comment]
Yeah, all weapon bullet were increased by 10. But it seems that the ammo type DOES affect the force that is applied to a prop.

http://wiki.garrysmod.com/page/Structures/AmmoData

**Comment by Kefta** [comment]
This is intended behaviour. FireBullets calculates weapon force by multiplying the ammo's force by phys_pushscale by the FireBullets "Force"/"m_flDamageForceScale " key. Maybe a flag could be added to skip ammo force but there's many other hardcoded behaviours that a FireBullets Lua rewrite would solve more effectively.

**Comment by ZehMatt** [comment]
Just came across this, as soon we supply the ammo type it will use its force and damage. Supplying a force on lets say "Pistol" ammo will scale it by that and the damage will be actually added. This should probably be documented on the Wiki.
